README.md

```diff
@@ -49,6 +52,9 @@ To specify additional logging driver options, you can use the `--log-opt NAME=VA
 | sumo-root-ca-path         | No        |               | Set the path to a custom root certificate.
 | sumo-server-name          | No        |               | Name used to validate the server certificate. By default, uses hostname of the `sumo-url`.
 | sumo-queue-size           | No        | 100           | The maximum number of log batches of size `sumo-batch-size` we can store in memory in the event of network failure, before we begin dropping batches. Thus in the worst case, the plugin will use `sumo-batch-size` * `sumo-queue-size` bytes of memory per container (default 100 MB).
+| tag                       | No        | {{.ID}}       | Specify a tag for messages, which interprets some markup. Default value is {{.ID}} (first 12 characters of the container ID). Refer to the [tag log-opt documentation] for customizing the log tag format.
```

Comment:
Would update wording to "Specifies a tag for messages, which can be used in the 'source category', 'source name', and source host' fields. Certain tokens of the form `{{X}}` are supported. Default value is `{{.ID}}`, the first 12 characters of the container ID. Refer to the [tag log-opt documentation] for more information and a list of supported tokens."